prototype/galore/test/test_downproj.py

```diff
@@ -0,0 +1,65 @@
+import logging
```

Comment:
note to self: boths tests pass but we need to decide whether we want benchmarks as tests or just accuracy checks